cmd/podmon/main.go

```diff
@@ -111,6 +111,9 @@ func main() {
 	if strings.Contains(*args.driverPath, "unity") {
```

Comment:
Disclaimer, I'm not a go dev, but... isn't it more readable to have an explicit switch/case than an if/else with fallback on powerflex ?

Reply:
Updated to switch case